### ISSUE TITLE
Deploying C6 failed due to psycopg2 version problem.

### ISSUE THREAD

**Issue by takabow** [takabow/cloudera-demo-env#14]
> Cloudera Manager 'First Run' command execution failed: Failed to perform First Run of services. ...

```
Thu Aug 30 17:47:34 JST 2018
JAVA_HOME=/usr/java/jdk1.8.0_121-cloudera
ERROR: Unable to find psycopg2 2.5.4 or higher version. You may need to manually install it. See http://tiny.cloudera.com/cdh6-hue-psycopg2
Thu Aug 30 17:47:36 JST 2018
JAVA_HOME=/usr/java/jdk1.8.0_121-cloudera
ERROR: Unable to find psycopg2 2.5.4 or higher version. You may need to manually install it. See http://tiny.cloudera.com/cdh6-hue-psycopg2
Thu Aug 30 17:47:39 JST 2018
JAVA_HOME=/usr/java/jdk1.8.0_121-cloudera
ERROR: Unable to find psycopg2 2.5.4 or higher version. You may need to manually install it. See http://tiny.cloudera.com/cdh6-hue-psycopg2
Thu Aug 30 17:47:43 JST 2018
JAVA_HOME=/usr/java/jdk1.8.0_121-cloudera
ERROR: Unable to find psycopg2 2.5.4 or higher version. You may need to manually install it. See http://tiny.cloudera.com/cdh6-hue-psycopg2
```

```
++ /usr/bin/python2.7 -c 'import psycopg2 ; print psycopg2.__version__.rsplit()[0]'
+ PSYCOPG2_VERSION=2.5.1
+ version_gt 2.5.4 2.5.1
++ tr ' ' '\n'
++ head -n 1
++ sort -rV
++ echo 2.5.4 2.5.1
+ test 2.5.4 == 2.5.4 -a 2.5.4 '!=' 2.5.1
+ echo 'ERROR: Unable to find psycopg2 2.5.4 or higher version. You may need to manually install it. See http://tiny.cloudera.com/cdh6-hue-psycopg2'
+ exit 1
```

**Comment by takabow** [comment]
https://www.cloudera.com/documentation/enterprise/6/latest/topics/cm_ig_extrnl_pstgrs.html#hue_postgres_install_psycopg2

![image](https://user-images.githubusercontent.com/66744/44843389-33875300-ac83-11e8-986f-7ca56c656e3b.png)

**Comment by takabow** [comment]
Still affected.

cloudera-scm-agent dpends on python-psycopg2, and version 2.5.1 is the latest on RHEL/CentOS 7 platform.

```
$ rpm -qa | grep psycopg2
python-psycopg2-2.5.1-3.el7.x86_64
```

```
 $ sudo yum erase python-psycopg2-2.5.1-3.el7.x86_64
Loaded plugins: fastestmirror
Resolving Dependencies
--> Running transaction check
---> Package python-psycopg2.x86_64 0:2.5.1-3.el7 will be erased
--> Processing Dependency: python-psycopg2 for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Running transaction check
---> Package cloudera-manager-agent.x86_64 0:6.0.0-530873.el7 will be erased
--> Finished Dependency Resolution

Dependencies Resolved

==============================================================================================================================================================================================================================================================================
 Package                                                                 Arch                                                    Version                                                             Repository                                                          Size
==============================================================================================================================================================================================================================================================================
Removing:
 python-psycopg2                                                         x86_64                                                  2.5.1-3.el7                                                         @base                                                              424 k
Removing for dependencies:
 cloudera-manager-agent                                                  x86_64                                                  6.0.0-530873.el7                                                    @cloudera-manager                                                  133 M

Transaction Summary
==============================================================================================================================================================================================================================================================================
Remove  1 Package (+1 Dependent package)

Installed size: 133 M
```

**Comment by takabow** [comment]
```
# rpm -qs python-psycopg2-2.5.1-3.el7.x86_64
normal        /usr/lib64/python2.7/site-packages/psycopg2
normal        /usr/lib64/python2.7/site-packages/psycopg2-2.5.1-py2.7.egg-info
normal        /usr/lib64/python2.7/site-packages/psycopg2/__init__.py
normal        /usr/lib64/python2.7/site-packages/psycopg2/__init__.pyc
normal        /usr/lib64/python2.7/site-packages/psycopg2/__init__.pyo
normal        /usr/lib64/python2.7/site-packages/psycopg2/_json.py
normal        /usr/lib64/python2.7/site-packages/psycopg2/_json.pyc
normal        /usr/lib64/python2.7/site-packages/psycopg2/_json.pyo
normal        /usr/lib64/python2.7/site-packages/psycopg2/_psycopg.so
normal        /usr/lib64/python2.7/site-packages/psycopg2/_range.py
normal        /usr/lib64/python2.7/site-packages/psycopg2/_range.pyc
normal        /usr/lib64/python2.7/site-packages/psycopg2/_range.pyo
normal        /usr/lib64/python2.7/site-packages/psycopg2/errorcodes.py
normal        /usr/lib64/python2.7/site-packages/psycopg2/errorcodes.pyc
normal        /usr/lib64/python2.7/site-packages/psycopg2/errorcodes.pyo
normal        /usr/lib64/python2.7/site-packages/psycopg2/extensions.py
normal        /usr/lib64/python2.7/site-packages/psycopg2/extensions.pyc
normal        /usr/lib64/python2.7/site-packages/psycopg2/extensions.pyo
normal        /usr/lib64/python2.7/site-packages/psycopg2/extras.py
normal        /usr/lib64/python2.7/site-packages/psycopg2/extras.pyc
normal        /usr/lib64/python2.7/site-packages/psycopg2/extras.pyo
normal        /usr/lib64/python2.7/site-packages/psycopg2/pool.py
normal        /usr/lib64/python2.7/site-packages/psycopg2/pool.pyc
normal        /usr/lib64/python2.7/site-packages/psycopg2/pool.pyo
normal        /usr/lib64/python2.7/site-packages/psycopg2/psycopg1.py
normal        /usr/lib64/python2.7/site-packages/psycopg2/psycopg1.pyc
normal        /usr/lib64/python2.7/site-packages/psycopg2/psycopg1.pyo
normal        /usr/lib64/python2.7/site-packages/psycopg2/tz.py
normal        /usr/lib64/python2.7/site-packages/psycopg2/tz.pyc
normal        /usr/lib64/python2.7/site-packages/psycopg2/tz.pyo
normal        /usr/share/doc/python-psycopg2-2.5.1
normal        /usr/share/doc/python-psycopg2-2.5.1/AUTHORS
normal        /usr/share/doc/python-psycopg2-2.5.1/LICENSE
normal        /usr/share/doc/python-psycopg2-2.5.1/NEWS
normal        /usr/share/doc/python-psycopg2-2.5.1/README
```

**Comment by takabow** [comment]
```
BEGIN sudo yum -y install cloudera-manager-agent
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: ftp-srv2.kddilabs.jp
 * epel: s3-mirror-ap-northeast-1.fedoraproject.org
 * extras: ftp-srv2.kddilabs.jp
 * updates: ftp-srv2.kddilabs.jp

Resolving Dependencies
--> Running transaction check
---> Package cloudera-manager-agent.x86_64 0:6.0.0-530873.el7 will be installed
--> Processing Dependency: cloudera-manager-daemons = 6.0.0 for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: httpd for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: fuse-libs for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: libxslt for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: openssl-devel for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: /lib/lsb/init-functions for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: bind-utils for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: cyrus-sasl-plain for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: cyrus-sasl-gssapi for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: MySQL-python for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: mod_ssl for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: perl for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64
--> Processing Dependency: python-psycopg2 for package: cloudera-manager-agent-6.0.0-530873.el7.x86_64 ★here!

**Comment by takabow** [comment]
Resolved.

Simply added python-psycopg2 from Postgres's official repos
https://github.com/takabow/cloudera-demo-env/blob/master/scripts/nonsecure-c6-cluster/bootstrap-common.sh#L8

**Comment by takabow** [comment]
fixed by ec1b6f08a27158c6d0a5238d0a9cb3800c491925
